### PR TITLE
[New Rule] Connection to Commonly Abused Web Services

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/475

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
**Matching Example:**

image

File: rules/windows/command_and_control_common_webservices.toml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2020/11/04"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may implement command and control communications that use common web services in order to hide their
```
Comment (peasead) [inline rules/windows/command_and_control_common_webservices.toml:10]:
```suggestion
Adversaries may implement command and control communications that use common web services in order to hide their
activity. This attack technique is typically targeted to an organization and uses web services common to the victim network which allows the adversary to blend into legitimate traffic.
```

**Review by peasead — APPROVED** [review]
Great query.

File: rules/windows/command_and_control_common_webservices.toml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                                          "*.file.io",
+                                          "*filebin.net",
+                                          "*slack-files.com",
+                                          "*ghostbin.*",
+                                          "*ngrok.*",
+                                          "*portmap.*",
+                                          "*serveo.net",
+                                          "*localtunnel.me",
+                                          "*pagekite.me",
+                                          "*localxpose.io"
```
Comment (dstepanic17) [inline rules/windows/command_and_control_common_webservices.toml:46]: Hey @Samirbous,

I saw this one ([notabug.org](https://www.virustotal.com/gui/domain/notabug.org/relations)) in the wild the other day and it ended up on this [write-up](https://blog.malwarebytes.com/cybercrime/2020/10/fake-covid-19-survey-hides-ransomware-in-canadian-university-attack/).  Might be worth to add to this already good list.

Comment (Samirbous) [reply rules/windows/command_and_control_common_webservices.toml:46]: nice!

**Review by dstepanic17 — APPROVED** [review]
LGTM

**Pull request merged**